applications/asset_tracker_v2/tests/json_common/src/json_validate.h

```diff
@@ -33,7 +33,8 @@
 						"\"v\":{"					\
 							"\"temp\":23,"				\
 							"\"hum\":50,"				\
-							"\"atmp\":101"				\
+							"\"atmp\":101,"				\
+							"\"airq\":50"				\
```

Comment:
Should only be included if sensor is enabled.

Reply:
Ill update the test to include the scenario where the air quality reading is disable.